dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/discovery/Navigation.java

```diff
@@ -118,15 +134,80 @@ public void addOptions(Options options) throws SAXException, WingException,
             }
         }
         browseGlobal.addItem().addXref(contextPath + "/community-list", T_head_all_of_dspace );
-    */
+        */
 
         /* regulate the ordering */
         options.addList("discovery");
         options.addList("browse");
         options.addList("account");
-        options.addList("context");
         options.addList("administrative");
-
+                
+        // get uri to see if using discovery and if under a specific handle
+        String uri = request.getSitemapURI();
+        
+        // check value in dspace.cfg
+        String search_export_config = ConfigurationManager.getProperty("xmlui.search.metadata_export"); 
+        
+        // get query
+        String query = decodeFromURL(request.getParameter("query"));
+                
+        // get scope, if not under handle returns null
+        String scope= request.getParameter("scope");
+        
+        // used to serialize all query filters together
+    	String filters = "";
+    	
+    	// get all query filters
+    	String[] fqs = DiscoveryUIUtils.getFilterQueries(ObjectModelHelper.getRequest(objectModel), context);
+        
+    	if (fqs != null)
+        {
+        	for(int i = 0; i < fqs.length; i++) {
+            	if(i < fqs.length - 1)
+            		filters += fqs[i] + ",";
+            	else
+            		filters += fqs[i];
+            }
+        }
+    	
+        if(uri.contains("discover")) {
```

Comment:
Instead of using this code, perhaps create a new Navigation class that is called only when the uri contains "discover", this can then be done in the sitemap.